src/component/Component.tsx

```diff
@@ -0,0 +1,21 @@
+import { Component, ReactNode } from 'react';
+
+export class Key extends Component {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
```

Comment:
Use componentDidMount to add a keyup handler, but it's missing componentWillUnmount to remove the handler.